pkg/option/config.go

```diff
@@ -2599,6 +2606,11 @@ func (c *DaemonConfig) AgentNotReadyNodeTaintValue() string {
 	}
 }
 
+// K8sNetworkPolicyEnabled returns false if cilium needn't support K8s NetworkPolicy
```

Comment:
Not sure why this describes the `false` case only. I'd say it returns whether it should be supported, or if you want true/false:

```suggestion
// K8sNetworkPolicyEnabled returns true if cilium agent needs to support K8s NetworkPolicy, false otherwise.
```